charts/substra-backend/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [24.3.01] - 2024-03-04
+
+### Added
+
+- Env var `BUILDER_ENABLED` in server
```

Comment:
why update this changelog ? The changes are not in the charts right ?

Reply:
Doc needs to be regenerated, I think? 🤔

Reply:
Because I forgot to do the change in the chart :), which is passing in the variable `BUILDERR_ENABLED` the value of `builder.enabled`

Reply:
Introduced in this commit: https://github.com/Substra/substra-backend/pull/847/commits/5fdd3504312920f01c663693f3130ef8c0ee754c thanks for your review 🙏